src/algorithms/acquisition/libs/acq_conf.cc

```diff
@@ -70,7 +70,7 @@ void Acq_Conf::SetFromConfiguration(const ConfigurationInterface *configuration,
     const int64_t fs_in_deprecated = configuration->property("GNSS-SDR.internal_fs_hz", fs_in);
     fs_in = configuration->property("GNSS-SDR.internal_fs_sps", fs_in_deprecated);
     doppler_max = configuration->property(role + ".doppler_max", doppler_max);
-    sampled_ms = configuration->property(role + ".coherent_integration_time_ms", sampled_ms);
+    sampled_ms = configuration->property(role + ".coherent_integration_time_ms", ms_per_code);
```

Comment:
Can you explain the purpose behind this change? Both values default to 1, and there is no parameter to change `ms_per_code`, so on first reading, I can't see that this does anything?